phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+  phpVersion: 70400
+  level: 0
+  paths:
+    - src
+    - the-events-calendar.php
+  scanDirectories:
+    - ../../../wp-includes

```

Comment:
Why are we scanning WP?

Reply:
My guess is so that the WP functions/methods/classes aren't shown as invalid. We can get around the need for scanning by including the following in composer.json as a dev dependency:

```
"szepeviktor/phpstan-wordpress": "^1.1",
```

I think doing that would avoid the need to download/install WP via wp-cli

Reply:
Updated to use the PHPStan WP stub library. My only concern is that the stubs stay up to date w/ the latest WP API (deprecations and new prototypes). Rare, but something we need to track over time.